recipes/nsls2-detector-handlers/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "nsls2-detector-handlers" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "08d632ef9e56155a2cabd4e1cca26671ed2461ad6d23552af0132d50e4ebb807" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - h5py
+    - numpy
+    - pandas
+
+test:
+  imports:
+    - nsls2_detector_handlers
+
+about:
+  home: https://github.com/bluesky/nsls2-detector-handlers
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Staging repo for handlers specific to NSLS-II
```

Comment:
That doesn't sound like a great description for the *package*

Reply:
I agree. I suggested a slight modification so it's a bit more self-explanatory.